### ISSUE TITLE
Double quotes matched across comment if followed by equal sign

### ISSUE THREAD

**Issue by otaithleigh** [atom/language-toml#18]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

When double quotes are followed by an equals sign in a comment on a line where there are already double quotes on the left hand side of the comment, highlighting breaks on that line. Does not occur for single quotes.

### Steps to Reproduce

1. Compare the results of the lines below in Atom:
```toml
b = "asdf"    # "asdf"
b = "asdf"    # "asdf" = 1
b = 'asdf'    # 'asdf' = 1
```

**Expected behavior:** Comments remain highlighted as comments

**Actual behavior:** Highlighting breaks on line with double quotes and equal sign

**Reproduces how often:** Always

### Versions

Windows 10 v1803 (64-bit)
```
Atom    : 1.26.1
Electron: 1.7.11
Chrome  : 58.0.3029.110
Node    : 7.9.0
apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.26.1
python 3.6.5
git 2.16.2.windows.1
visual studio
```

### Additional Information

N/A

**Comment by Arcanemagus** [comment]
For reference, this is how the above is rendered in `1.31.0-nightly9`:
![image](https://user-images.githubusercontent.com/427137/44387229-c5e86380-a4d9-11e8-8a92-e893febfa568.png)

**Comment by 50Wliu** [comment]
Fixed by #21 :tada:
![double-quotes-comment-equals](https://user-images.githubusercontent.com/2766036/54896724-e6c3cd80-4e9b-11e9-8002-59124ca3997b.png)
